tasks/options/postcss.js

```diff
@@ -11,7 +11,7 @@ module.exports = function(config) {
       },
 
       processors: [
-        require('autoprefixer')({browsers: 'last 3 versions'}), // add vendor prefixes
+        require('autoprefixer')({browsers: ['last 3 versions', 'safari >= 4']}), // add vendor prefixes
```

Comment:
yes, but not sure I want add add this to prefixer, unsure what that will do as it's such an old browser .